integration-cli/trust_server.go

```diff
@@ -106,7 +106,7 @@ func newTestNotary(c *check.C) (*testNotary, error) {
 	}
 
 	// Wait for notary to be ready to serve requests.
-	for i := 1; i <= 5; i++ {
+	for i := 1; i <= 20; i++ {
```

Comment:
I agree. But even after 20 attempts, that is still a 200 ms timeout. I'd suggest raising it even further. It is very possible for I/O or resource starvation to make launching a service take more than 200 ms.